docs/authentication/configuration/restrictions.mdx

```diff
@@ -1,91 +1,79 @@
 ---
-title: Restrictions | Authentication
+title: Restrictions
 description: Learn how to manage user access to your application by configuring Clerk's access restrictions.
 ---
 
 # Restrictions
 
-Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Through these restriction settings, you can limit sign-ups or prevent accounts with specific identifiers from accessing your application. These identifiers can be email addresses, phone numbers, and even entire domains. To get started, in the Clerk Dashboard, go to **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)**.
+Clerk provides a set of restriction options designed to provide you with enhanced control over who can gain access to your application. Through these restriction settings, you can limit sign-ups or prevent accounts with specific identifiers from accessing your application. These [identifiers](/docs/authentication/configuration/sign-up-sign-in-options#identifiers) can be email addresses, phone numbers, and even entire domains.
+
+There are two main types of restrictions available:
+
+- [Allowlist](#allowlist) - Allows only specific identifiers to sign up for your application.
+- [Blocklist](#blocklist) - Blocks specific identifiers from signing up for your application.
+
+In addition to these, there is also the [Block email subaddresses](#block-email-subaddresses) feature, which blocks email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts.
+
+All of these restrictions can be enabled and used together to provide a more secure and controlled environment for your application. In the case you have enabled the allowlist and the blocklist and have added the same identifier in both, the allowlist takes precedence.
```

Comment:
>  In the case you have enabled the allowlist and the blocklist and have added the same identifier in both, the allowlist takes precedence.

Nice, I know this is mentioned in the Dashboard too but this is a good one to make sure people **know**